src/resources/postcss/freemius.pcss

```diff
@@ -0,0 +1,227 @@
+.events-cal,
+.toplevel_page_tribe-common,
+.toplevel_page_tribe-help,
+.toplevel_page_tribe-app-shop {
+	#fs_connect {
+		background-color: #ffffff;
+		margin: 20px 20px 0 2px;
+		max-width: 1200px;
+		position: relative;
+		width: auto;
+
+		.fs-visual {
+			align-content: center;
+			align-items: center;
+			background-color: transparent;
+			display: flex;
+			margin: 0 35px;
+			padding: 20px 0 0;
+
+			.fs-site-icon,
+			.fs-plugin-icon,
+			.fs-first.dashicons-plus,
+			.fs-second.dashicons-plus,
+			.fs-connect-logo {
+				border: 0;
+				bottom: auto;
+				left: auto;
+				margin: 0;
+				position: relative;
+				right: auto;
+				top: auto;
+				width: auto;
+			}
+
+			.fs-site-icon,
+			.fs-plugin-icon {
+				width: 60px;
+			}
+
+			.dashicons-plus {
+				color: #17a0d6;
+				padding-left: 4px;
+				padding-right: 4px;
+				width: auto;
+			}
+
+			.fs-site-icon {
+				.dashicons {
+					visibility: hidden;
+				}
+
+				&::before {
+					background-image: url( ../images/event-tickets.png );
```

Comment:
@cliffordp Let's get the new SVG in place

Reply:
We can delete the png that we added as part of this PR

Reply:
https://ps.w.org/event-tickets/assets/icon.svg

Reply:
done